portfolio.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+require 'admin/functions.php';
+
+$db = connectDatabase();
+
+$stmt = $db->query(
+    "SELECT `id`,`title`, `mini_description`, `background_image`, `project_url` 
+    FROM `projects`;");
+$projects_data = $stmt->fetchAll();
+
+$stmt2 = $db->query(
+    "SELECT `id`, `bio_title`, `bio`, `contact_title`, `email`, `telephone`
+    FROM `about_me`;");
+$about_me_data = $stmt2->fetchAll();
+
+$stmt3 = $db->query(
+    "SELECT `id`, `name`, `sub_title`, `tagline`
+    FROM `content`;");
+$content_data = $stmt3->fetchAll();
+
+if (count($projects_data) % 2 !== 0) {
```

Comment:
id comment this to say why your doing it

Reply:
done